pkgs/development/python-modules/libcst/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, typing-inspect
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "libcst";
+  version = "0.3.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1zgwxdbhz2ljl0yzbrn1f4f464rjphx0j6r4qq0csax3m4wp50x1";
+  };
+
+  propagatedBuildInputs = [ typing-inspect pyyaml ];
+
+  # Test fails with ValueError: No data_provider tests were created for
+  # test_type_availability! Please double check your data.
+  # The tests appear to be doing some dynamic introspection, not sure what is
+  # going on there.
+  doCheck = false;
```

Comment:
Could you at least add a `pythonImportsCheck` then so that this package isn't completely untested?

Reply:
I did not know about `pythonImportsCheck`, thanks for pointing this out! Added.